### DISCUSSION TITLE
Use an A100 and train my own pixArt-δ in fewer days

### DISCUSSION THREAD

**Discussion by zmy-08** [PixArt-alpha/PixArt-alpha#117] [Q&A]
**It's a great honor to use such a great job, and I have a question I would like to ask you.**  **The following is the original sentence from the paper:** _“Additionally, PIXART-δ is designed to be efficiently trainable on 32GB V100 GPUs within a single day.With its 8-bit inference capability (von Platen et al., 2023), PIXART-δ can synthesize 1024px images within 8GB GPU memory constraints, greatly enhancing its usability and accessibility.”_  **Does this mean that I use an A100 and can train my own pixArt-δ in fewer days?**

**Comment by lawrence-cj** [comment]
Yes.
